### PR TITLE
fixing pt and pt_BR translations (related to #22)

### PR DESCRIPTION
**Pull request by arademaker**
This PR is a response to issue #22 . There are many more problems in the "pt" translations. I would vote to remove the pt_BR variation and keep only one pt for European and Brazilian Portuguese. 

Sorry for being late in the response of the issue. Hope that this PR can be accepted. If so, I can keep doing more fixes.

### CODE REVIEW

**Comment by RicardoUsbeck** [comment]
Hi @arademaker thanks for your pull request! 

What is your rationale behind remove the pt_BR variation?

Dear @DiegoMoussallem could you take a look at this PR since I cannot judge it?

**Comment by DiegoMoussallem** [comment]
Hi @RicardoUsbeck, I will check it. @arademaker nice contribution ;)

**Comment by arademaker** [comment]
@RicardoUsbeck, many language resources do not take the variations as different enough. Take the UD corpora http://github.com/universaldependencies/UD_Portuguese-Bosque. It contains half of the sentences in European and half in BR Portuguese. This is the most revised  PT corpus for training universal dependencies. Any parser that uses this corpus to train is learning European and BR Portuguese. Another example is our http://openwordnet-pt.org, the most complete Wordnet for Portuguese. We embrace the two variations although most of the contributors are Brazilians. 

To sum up, I don't believe that for this task, the two PT variations would make sense. Moreover, I found too few cases with PT_BR, and most of the questions are marked with only PT.

Does it make sense?

**Comment by arademaker** [comment]
@DiegoMoussallem, does it make sense to continue the revision? There are more questions marked with `PT` that are weird translations.

**Comment by RicardoUsbeck** [comment]
@arademaker makes total sense to me

**Comment by arademaker** [comment]
If the question is for me, the answer is yes! ;-) Just waiting you guys decide if it makes sense to invest more time on the revision of the remain PT questions...

**Comment by DiegoMoussallem** [comment]
@arademaker I agree with your point of view. For this benchmarking, the important variations of PT_PT and PT_BR are not taken into account, and also the task does not consider it. So, it makes totally sense to remove PT-BR. Please do not invest more time on PT_BR, let's keep everything as PT and that's it. I will check your contribution @arademaker  and also include mine there.

**Comment by arademaker** [comment]
So, it is just a matter of replace "pt_BR" with "pt" in the `language` of the questions. In that way, we will not lose examples. To avoid conflicts in the files, I will wait for the merge of this PR and the changes you just mention you have. After that, please use #22 to alert me and I can make a double check in the translations again. Does it work?

**Comment by DiegoMoussallem** [comment]
Let's do the following to avoid all kinds of conflicts in the files which are really boring to fix, first, we merge your pull request then we work on it with some of my changes and new ones from your side. @RicardoUsbeck please merge his pull request.

**Comment by arademaker** [comment]
But wasn’t that my proposal? ;-) 

The only thing is that if you have changes to apply, it is better you apply them after this merge, submit a new PR. I will make new round of revision only after your changes were accepted.

**Comment by DiegoMoussallem** [comment]
yep, I meant about my changes, lol because I do not have write access, so it would be another pull request
